### PR TITLE
Add camel-fop wrapper

### PR DESCRIPTION
**Pull request by lopushen**
Added `fop-core` dependency with wrap protocol, exporting only required packages, as there is no servicemix bundle (only 2.7 present, but 2.9 needed)

### CODE REVIEW

File: features/src/main/feature/camel-features.xml [hunk 1]
```diff
@@ -749,33 +749,11 @@
...
-<!--        <bundle dependency="true">wrap:mvn:org.apache.xmlgraphics/batik-ext/${xmlgraphics-batik-version}$Bundle-SymbolicName=org.apache.xmlgraphics.batik-ext&amp;Bundle-Version=${xmlgraphics-batik-version}</bundle>-->
-<!--        <bundle dependency="true">wrap:mvn:xml-apis/xml-apis-ext/${xml-apis-ext-version}$Bundle-SymbolicName=xml-apis-ext&amp;Bundle-Version=${xml-apis-ext-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.bcel/${bcel-bundle-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.fop/${fop-bundle-version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-fop/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-fop' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <bundle dependency='true'>wrap:mvn:org.apache.xmlgraphics/fop-core/2.9$Export-Package=org.apache.fop.apps;version=2.9,org.apache.fop.pdf;version=2.9</bundle>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]: Please use the variable `fop-version`

Comment (lopushen) [reply features/src/main/feature/camel-features.xml]: fixed

File: features/src/main/feature/camel-features.xml [hunk 2]
```diff
@@ -722,33 +722,11 @@
...
-<!--        <bundle dependency="true">wrap:mvn:org.apache.xmlgraphics/batik-ext/${xmlgraphics-batik-version}$Bundle-SymbolicName=org.apache.xmlgraphics.batik-ext&amp;Bundle-Version=${xmlgraphics-batik-version}</bundle>-->
-<!--        <bundle dependency="true">wrap:mvn:xml-apis/xml-apis-ext/${xml-apis-ext-version}$Bundle-SymbolicName=xml-apis-ext&amp;Bundle-Version=${xml-apis-ext-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.bcel/${bcel-bundle-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.fop/${fop-bundle-version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-fop/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-fop' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <bundle dependency='true'>wrap:mvn:org.apache.xmlgraphics/fop-core/${fop-version}$Export-Package=org.apache.fop.apps;version=${fop-version},org.apache.fop.pdf;version=${fop-version}</bundle>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]: Why do you want to export only those packages?

Comment (lopushen) [reply features/src/main/feature/camel-features.xml]: There are over 20 packages in fop-core and only these 2 are required (and the original commented out xml suggested that). Also works without the export statement as well

Comment (essobedo) [reply features/src/main/feature/camel-features.xml]: I don't see anything in the original feature that could suggest it, indeed this dependency was initially wrapped by the SMX bundle `org.apache.servicemix.bundles.fop`, and if you look at the manifest you will see that all packages are exported.  

I would rather prefer to keep a consistent way to wrap dependencies and so far we have never limited the exported packages, so if you don't mind let's keep it simple and remove the export package section

Comment (lopushen) [reply features/src/main/feature/camel-features.xml]: I meant the commented out dependency, but the smx bundle exports `org.apache.fop` in full, so fixed. Thanks!

**Review by essobedo — APPROVED** [review]
LGTM

**Pull request merged**